src/lib/input/autosize.ts

```diff
@@ -34,4 +34,12 @@ export class MatTextareaAutosize extends CdkTextareaAutosize {
   @Input()
   get matAutosizeMaxRows(): number { return this.maxRows; }
   set matAutosizeMaxRows(value: number) { this.maxRows = value; }
+
+  @Input('mat-autosize')
```

Comment:
Is this the correct name, @mmalerba?

Reply:
Should be, see the selector definition: `selector: 'textarea[mat-autosize], textarea[matTextareaAutosize]',`

Reply:
Aha, yes :)